teps/0096-pipelines-v1-api.md

```diff
@@ -123,19 +131,59 @@ of TEP-0033: Feature Gates, Tekton's policy allows a behavior change to be updat
 A behavior can't be changed from opt-in to opt-out until 9 months after it is introduced to a beta API or 12 months after it is introduced to a stable API.
 This is different from Kubernetes' policy, which would require creating a new API version to change a feature from opt-in to opt-out.
 
-### API Definition
-
 ## Use Cases
 
-### V1 Supported Use Cases
+Most Tekton use cases are supported by features from multiple Tekton projects, meaning that there are relatively few use cases for
+Pipelines as an independent project. The following use cases guide Tekton feature work, and a Pipelines V1 API should stabilize
+the features critical to supporting them.
+
+### Use Cases prioritized for Tekton
+- As an application engineer, I want to run tests when new pull requests are created and automatically
+merge only changes that have passed these tests, so we are always in a releaseable state.
+- As an application engineer, I want to be able to build, publish, and deploy my application to staging and prod environments.
+-  As a platform engineer, I want to provide functionality that’s not directly available in the Tekton Pipelines API to application engineers in my organization.
+
+### Use Cases not prioritized
+The following use cases may currently be supported by Tekton, but features that solely address these use cases should not be
+V1 API blockers.
+
+- Non-CI/CD workflows, such as machine learning use cases
```

Comment:
I don't see this explicit reference to "machine learning" necessary here. This could create conflicts since it's explicitly spelled out. Machine learning is a huge space and Tekton Pipelines is a CI/CD platform. Just having it listed as non-CI/CD workflows is enough.

Reply:
done